src/Bridges/intervalbridge.jl

```diff
@@ -48,6 +48,26 @@ function MOI.get(model::MOI.ModelLike, a::MOI.ConstraintDual, c::SplitIntervalBr
     return lower_dual > -upper_dual ? lower_dual : upper_dual
 end
 
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintBasisStatus, c::SplitIntervalBridge)
+    lower_stat = MOI.get(model, MOI.ConstraintBasisStatus(), c.lower)
+    upper_stat = MOI.get(model, MOI.ConstraintBasisStatus(), c.upper)
+    if lower_stat == MOI.NONBASIC_AT_LOWER || upper_stat == MOI.NONBASIC_AT_UPPER
+        Compat.@warn("Single sided constraints should not have basis status:" *
+        " NONBASIC_AT_LOWER or NONBASIC_AT_UPPER, instead use NONBASIC")
```

Comment:
It's probably more helpful to have a different warning for each case so that the user sees in which case he is